### ISSUE TITLE
Ctrl+F can't find word in languages except English

### ISSUE THREAD

**Issue by oceancx** [janisdd/vscode-edit-csv#11]

**Comment by janisdd** [comment]
Actually I did nothing to Ctrl+F to work so I guess it's vs code default search?
From my testing Ctrl+F does only find visible text e.g. when you have 1000 rows and only see 1-50 and want to find something in row 51-1000
This is because the virtual renderer of handsontable (not visible rows are not rendered).

I can look into that but I'm not sure if this is possible

**Comment by oceancx** [comment]
Ctrl+F search only visible words is too limit...I have written a .tsv editor using ImGUI also face the similar issue, but it's easy to solve using ImGUI.

**Comment by janisdd** [comment]
I agree, a full search would be nice. I looked through the handsontable docs and they support this...
So this is a matter of time

**Comment by oceancx** [comment]
that will be great~

**Comment by janisdd** [comment]
I've made some progress on this...
![findWidget](https://user-images.githubusercontent.com/12775963/67112029-96fd7880-f1d6-11e9-95d1-9520aae11430.png)
I stayed close to the vs code find so that users should be familiar with it.
However, this is not finished and there are some things I need to figure out
e.g. what does the regex+whole word option do (together)?

**Comment by oceancx** [comment]
thank u for this improvement. I almost never use the whole word option to find text. I don't know what they do together too...

**Comment by janisdd** [comment]
I think together they should match when the regex selects the whole cell value (even if this is not what vs code does).
It will take some more time to finish this up... (get all the behaviour right)

**Comment by oceancx** [comment]
when the cell value contains space ,should they match too?

**Comment by janisdd** [comment]
I thought about that too but I'm still not sure what is more useful. Maybe an additional toggle button (next to the button `Match whole cell value`) might be useful: `Match trimmed cell value` (instead of using a regex)

**Comment by janisdd** [comment]
Version 2.2 includes the new find widget
